string base
---
bool success
string message

